src/utils/vaults/LiquidatedVaultDerivedValues.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  LoanVaultActive,
+  LoanVaultLiquidated,
+  LoanVaultState,
+  LoanVaultTokenAmount
+} from '@defichain/whale-api-client/dist/api/loan'
+import { LiquidatedVaultDerivedValues } from '../../pages/vaults'
+import BigNumber from 'bignumber.js'
+
+export function calculateLiquidationValues (vault: LoanVaultActive | LoanVaultLiquidated): LiquidatedVaultDerivedValues | undefined {
+  const loanTokens: LoanVaultTokenAmount[] = []
+  const collateralTokens: LoanVaultTokenAmount[] = []
+  let totalLoanValue = new BigNumber(0)
+  let totalCollateralValue = new BigNumber(0)
+  let totalCollateralRatio = new BigNumber(0)
+
+  let isMissingActivePrice = false
+
+  if (vault.state === LoanVaultState.IN_LIQUIDATION) {
+    vault.batches.forEach(batch => {
+      if (batch.loan?.activePrice?.active == null) {
+        if (batch.loan.id === '15') {
```

Comment:
on TestNet/Playground, it ID will be different, maybe use `DUSD` symbol instead to check?